src/06-time-tracking/time-tracking.apib

```diff
@@ -172,3 +172,14 @@ Start a new timer based on previously tracked time.
         + data (object)
             + type: `timer` (string)
             + id: `f01adf4a-bb9b-45de-b231-615cd0e941de` (string)
+
+### timeTracking.delete [GET /timeTracking.delete]
```

Comment:
Not really B but also not PP: `POST /timeTracking.delete`.

`GET` works but it's a request modifying or deleting data, and we always document with `POST` for that.

Reply:
What @dietrichm said!